apps/public-docsite-v9/.storybook/manager-head.html

```diff
@@ -108,7 +108,8 @@
     color: #ffffff !important;
   }
 
-  .sidebar-subheading {
+  .sidebar-subheading,
+  button[data-action='collapse-ref'] {
```

Comment:
To ensure the styles of the `Contributors Packages` menu item are consistent with the other items in the sidebar, the `button[data-action='collapse-ref']` is the best selector I could identify.

<img width="1251" alt="Screenshot 2024-11-01 at 13 20 38" src="https://github.com/user-attachments/assets/6c7a6655-c8a6-4ed0-8083-47a50ac82548">

Reply:
can we add this intel to the source via comment ? ty